actors/builtin/miner/miner_state.go

```diff
@@ -259,6 +269,151 @@ func (st *State) ComputeProvingSet(store adt.Store) ([]abi.SectorInfo, error) {
 	return sectorInfos, nil
 }
 
+func computeVestAmount(total abi.TokenAmount, cliffStartEpoch abi.ChainEpoch, fullyVestedEpoch abi.ChainEpoch, quantizedEpoch abi.ChainEpoch) abi.TokenAmount {
+	// PARAM_FINISH linear vesting placeholder
+	numVestWindows := (fullyVestedEpoch - cliffStartEpoch) / VestIncrement
+	_ = (quantizedEpoch - cliffStartEpoch) / VestIncrement // number of VestIncrement elapsed for other vest functions
+	quantizedVestAmount := big.Div(total, big.NewInt(int64(numVestWindows)))
+	return quantizedVestAmount
+}
+
+func getQuantizedEpoch(e abi.ChainEpoch) abi.ChainEpoch {
+	// PARAM_FINISH
+	// default is the nearest next epoch that is multiple of QuantizedGranularity after a VestIncrement offset
+	// computeVestAmount can assume at least one VestIncrement has elapsed
+	offsetEpoch := e + VestIncrement
```

Comment:
Why is VestIncrement used here at all? I expect quantization to ... just quantize. 

If `e` is an exact multiple of the QuantizedGranularity, this should return `e`.